src/System.Net.Security/tests/UnitTests/project.json

```diff
@@ -10,6 +10,10 @@
     "System.Security.Cryptography.X509Certificates": "4.4.0-beta-24613-01",
     "System.Text.RegularExpressions": "4.4.0-beta-24613-01",
     "System.Resources.ResourceManager": "4.4.0-beta-24613-01",
+    "System.Net.Security": {
```

Comment:
That's exactly what this is doing. Even though System.Net.Security wasn't directly referenced before, it was getting pulled in as a nested dependency, but with explicitly adding it here and having the `exclude: compile` line, we are removing it even when it get's pulled in by other dependencies.